_authors/tiffanylamm.markdown

```diff
@@ -0,0 +1,7 @@
+---
+name: Tiffany Lam 
+short_name: tifflam
+github: tiffanylamm
+---
+
+**<Tiffany Lam>** <I am a junior at UC Berkeley!>
```

Comment:
```suggestion
**Tiffany Lam** I am a junior at UC Berkeley!
```
Can you make this change?

Reply:
got it. i'll move it over.

Reply:
@ashwin-pc I've made the above changes. Do I need to make new PR?

Reply:
@tiffanylamm If you make your changes on the same branch and then `git push`, it should update this PR automatically.

Reply:
@tiffanylamm hey, wanted to checkin and remind you to push your changes so we can get this merged!

Reply:
I've pushed my changes. Sorry for the hold up I thought pushed it correctly but I didn't.